helm/charts/hapi-fhir/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
@manutarus should these be moved to the charts directory so that they're ignored by default for all charts?